tests/Find-DbaInstance.Tests.ps1

```diff
@@ -24,9 +24,9 @@ Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
     Context "Command finds SQL Server instances" {
         BeforeAll {
             if ($env:APPVEYOR) {
-                $results = Find-DbaInstance -ComputerName $script:instance3 -ScanType Browser, SqlConnect
+                $results = Find-DbaInstance -ComputerName $script:instance3 -ScanType Browser, SqlConnect | Select-Object -First 1
```

Comment:
I'd prefer this be filtering on the actual `$instance3` database service and not randomly picking whatever comes first in the list.

If we do that then the whole if/else block can be removed because it would match no matter the environment.